packages/wasm/package.json

```diff
@@ -16,11 +16,11 @@
     "test:rust": "cd crate ; wasm-pack test --headless --firefox -- --test build --target wasm32-unknown-unknown --release --features 'mock-database'"
   },
   "dependencies": {
-    "bech32": "^2.0.0",
-    "zod": "^3.22.4",
-    "@penumbra-zone/types": "workspace:*"
+    "@penumbra-zone/types": "workspace:*",
+    "zod": "^3.22.4"
   },
   "devDependencies": {
+    "@penumbra-zone/bech32": "workspace:*",
```

Comment:
bech32 is used in runtime, probably should go up to normal deps (unless we are only using the types)

Reply:
it's only used in a test, which is not runtime, it belongs in devdeps